### PR TITLE
change names of fields to make code eaiser to follow

### PR DESCRIPTION
**Pull request by mhilton**

### CODE REVIEW

File: httpbakery/form/form.go [hunk 1]
```diff
@@ -99,15 +99,15 @@ type SchemaResponse struct {
...
 
-// LoginRequest is a request to perform a login using the provided details.
+// LoginRequest is a request to perform a login using the provided form.
 type LoginRequest struct {
 	httprequest.Route `httprequest:"POST"`
 	Login             LoginWrapper `httprequest:",body"`
 }
 
-// Login holds the body of a login request.
+// LoginWrapper holds the body of a form login request.
```
Comment (rogpeppe) [inline httpbakery/form/form.go]: LoginBody might be a better name. LoginRequest.Body.Form ?

File: httpbakery/form/form.go [hunk 2]
```diff
@@ -99,15 +99,15 @@ type SchemaResponse struct {
...
 type LoginWrapper struct {
-	Login map[string]interface{} `json:"login"`
+	Form map[string]interface{} `json:"login"`
```
Comment (rogpeppe) [inline httpbakery/form/form.go]: The JSON field name should probably match the actual field name here.

**Comment by rogpeppe** [comment]
LGTM with a couple of suggestions.

File: httpbakery/form/form.go (hunk 1 above)
Comment (mhilton) [inline httpbakery/form/form.go]: done

File: httpbakery/form/form.go (hunk 2 above)
Comment (mhilton) [inline httpbakery/form/form.go]: done

**Pull request merged**